app/main.py

```diff
@@ -1,16 +1,23 @@
 class SoftwareEngineer:
-    skills = []
 
-    def __init__(self, name: str):
+    def __init__(self, name: str, skills=[]):
         self.name = name
+        self.skills = skills
+        if isinstance(self, FrontendDeveloper):
+            self.skills = ["JavaScript", "HTML", "CSS"]
+        if isinstance(self, BackendDeveloper):
+            self.skills = ["Python", "SQL", "Django"]
+        if isinstance(self, AndroidDeveloper):
+            self.skills = ["Java", "Android studio"]
+        if isinstance(self, FullStackDeveloper):
+            self.skills = ["Python", "SQL", "Django",
+                           "JavaScript", "CSS", "HTML"]
```

Comment:
you should define `self.skills = ["Python", "SQL", "Django"]` in `BackendDeveloper` constructor, not in `SoftwareEngineer`, also other cases